requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+coverage==4.4.1
+py==1.4.34
+pytest==3.1.1
```

Comment:
This dependency is specified in `requirements.txt`. Don't duplicate dependencies.